openhabian-setup.sh

```diff
@@ -75,6 +75,8 @@ if [[ -n "$UNATTENDED" ]]; then
   srv_bind_mounts
   permissions_corrections
   misc_system_settings
+# not per default for now
+# if is_pione || is_pitwo || is_pithree || is_pithreeplus || is_pine64; then init_zram_mounts install; fi
```

Comment:
is_pifour also available, you may add it to avoid problems once becomes active

Reply:
Good advice (Y)

Reply:
added is_pifour()